dm_cli/import_entity.py

```diff
@@ -39,16 +50,47 @@ def import_single_entity(path: Path, path_reference: str):
             dependencies,
             data_source_id,
             file_path=package,
-            parent_directory=path.parent,
+            parent_directory=source_path.parent,
         )
 
     # Upload blobs
     prepared_document = upload_blobs_in_document(prepared_document, data_source_id)
     document_json_str = json.dumps(prepared_document)
-
     dmss_api.document_add_to_path(
-        f"{path_reference}",
+        f"{destination}",
         document_json_str,
         update_uncontained=True,
         files=[],
     )
+
+
+def import_folder_entity(source_path: Path, destination: str, overwrite: bool = False) -> None:
+    click.echo(f"Importing PACKAGE '{source_path.name}' --> '{destination}/'")
+    destination_path = Path(destination)
+    data_source = destination_path.parts[0]
+
+    ensure_package_structure(destination_path)
+
+    try:
+        dmss_api.document_get_by_path(f"dmss://{destination}/{source_path.name}")
+        if not overwrite:
+            raise ValueError(f"Failed to upload to 'dmss://{destination}/{source_path.name}' - It already exists.")
```

Comment:
didn't quite understand this - do we know here that 'dmss://{destination}/{source_path.name}' exist?
I would assume we need to extend the `if` to check the response from dmss_api.document_get_by_path()

`if not overwrite and .....`:

Reply:
The generated python API used to get the document will raise an exception if it is not found.
Therefore, if no exception. It must exist